languages/ruby/lib/oso/polar/errors.rb

```diff
@@ -100,6 +100,9 @@ class UnrecognizedToken < ParseError; end
     class ApiError < Error; end
     class ParameterError < ApiError; end
 
+    class ValidationError < Error; end
```

Comment:
@gj do you see any reason to keep this `ValidationError` type around?

Reply:
Nope